c7n/resources/account.py

```diff
@@ -1731,3 +1731,76 @@ def process(self, resources):
         client.put_block_public_access_configuration(
             BlockPublicAccessConfiguration=updatedConfig
         )
+
+
+@filters.register('sechub-enabled')
+class SecHubEnabled(Filter):
+    """Filter an account depending on whether security hub is enabled or not.
+
+    :example:
+
+    .. code-block:: yaml
+
+       policies:
+         - name: check-securityhub-status
+           resource: aws.account
+           filters:
+            - type: sechub-enabled
+              state: true
+
+    """
+
+    permissions = ('securityhub:DescribeHub',)
+
+    schema = type_schema('sechub-enabled', state={'type': 'boolean'})
+
+    def process(self, resources, event=None):
+        state = self.data.get('state', True)
+        client = local_session(self.manager.session_factory).client('securityhub')
+        try:
+            sechub = client.describe_hub()
+        except ClientError as e:
+            if e.response['Error']['Code'] == 'InvalidAccessException':
```

Comment:
this needs a comment for the case it represents (region not activated)

minor, this try/except can be replaced with a call to self.manager.retry with the ignorable error codes.

Reply:
Wrapped self.manager.retry inside try/except to make sure we catch `InvalidAccessException`. Thanks for the suggestion!

Reply:
so the latest versions of trunk these error codes can be given directly to retry, instead of a try/except.

Reply:
ah, got it! Didn't know about that, modified code to use that. Thanks!